passt/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:latest
```

Comment:
please use a ubi image instead since it's similar to what we have downstreawm but it works upstream.